fastbreak/src/main/java/com/nike/fastbreak/CircuitBreaker.java

```diff
@@ -231,6 +236,13 @@ ET executeBlockingCall(Callable<ET> eventSupplier)
     CircuitBreaker<ET> onOpen(Runnable listener);
 
     /**
+     * Identifier of the `CircuitBreaker` instance. This is implementation specific and there is no guarantee of uniqueness.
+     */
```

Comment:
I'd add one final sentence to this javadoc: `If not overridden by the implementation then {@link #DEFAULT_ID} will be returned.`.

Reply:
done!